NetCore/Resize.cs

```diff
@@ -122,24 +126,28 @@ public SKSize SkiaCanvasResizeBenchmark()
         public SKSize SkiaBitmapResizeBenchmark()
         {
             using (var original = new SKBitmap(Width, Height))
-            using (var resized = original.Resize(new SKImageInfo(ResizedWidth, ResizedHeight), SKFilterQuality.High))
+            using (var resized = original.Resize(new SKImageInfo(ResizedWidth, ResizedHeight), SKFilterQuality.Medium))
             using (var image = SKImage.FromBitmap(resized))
             {
                 return new SKSize(image.Width, image.Height);
             }
         }
 
         [Benchmark(Description = "NetVips Resize")]
-        public (int width, int height) NetVipsResize()
+        public (int width, int height, byte[] memory) NetVipsResize()
         {
             // Scaling calculations
             const double xFactor = (double)Width / ResizedWidth;
             const double yFactor = (double)Height / ResizedHeight;
 
-            using (var original = NetVips.Image.Black(Width, Height).CopyMemory())
-            using (var resized = original.Reduce(xFactor, yFactor, kernel: Enums.Kernel.Cubic).CopyMemory())
+            using (var original = NetVips.Image.Black(Width, Height))
+            using (var resized = original.Resize(1.0 / xFactor, vscale: 1.0 / yFactor, kernel: Enums.Kernel.Cubic))
             {
-	            return (resized.Width, resized.Height);
+                // libvips is "lazy" and will not process pixels
+                // until you write to an output file, buffer or memory
+                var memory = resized.WriteToMemory();
+
+                return (resized.Width, resized.Height, memory);
```

Comment:
Note that I returned the memory here to avoid dead code elimination: https://benchmarkdotnet.org/articles/guides/good-practices.html#avoid-dead-code-elimination

Reply:
I see, resolved within: 8fbc3c7484967f54931652623ef9e44131834992.